ql/src/java/org/apache/hadoop/hive/ql/parse/repl/dump/events/AbstractEventHandler.java

```diff
@@ -71,4 +85,31 @@ public long fromEventId() {
   public long toEventId() {
     return event.getEventId();
   }
+
+  public void writeFileEntry(String dbName, Table table, String file, BufferedWriter fileListWriter,
+                             Context withinContext) throws IOException, LoginException {
+    HiveConf hiveConf = withinContext.hiveConf;
+    String distCpDoAsUser = hiveConf.getVar(HiveConf.ConfVars.HIVE_DISTCP_DOAS_USER);
+    if (Utils.shouldDumpMetaDataOnly(table, withinContext.hiveConf)) {
```

Comment:
If its metadata only, why are we dumping data

Reply:
Will fix this.